lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
```

Comment:
Not sure about the naming here
EventSource is used to receive server-sent events
The function in question is emitting server-sent events